scripts/docs/generate_docc.sh

```diff
@@ -47,30 +47,9 @@ declare -r build_path_linux="$build_path/"
 declare -r docc_source_path="$root_path/.build/swift-docc"
 declare -r docc_render_source_path="$root_path/.build/swift-docc-render"
 
-# Prepare and build docc
-if [[ ! -d "$docc_source_path" ]]; then
-  git clone https://github.com/apple/swift-docc.git "$docc_source_path"
-  cd $docc_source_path
-
-  if [[ ! -d "$docc_source_path/$build_path" ]]; then
-    swift build -c release
-  fi
-else
-  echo "Assuming docc is built..."
-fi
-
-
-if [[ ! -d "$docc_render_source_path" ]]; then
-  git clone https://github.com/apple/swift-docc-render.git "$docc_render_source_path"
-  cd $docc_render_source_path
-
-  npm install
-  npm run build
-else
-  echo "Assuming docc-render is built..."
-fi
-
-export DOCC_HTML_DIR=$docc_render_source_path/dist
+# Accessing docc from toolchain
+export TOOLCHAIN=/Library/Developer/Toolchains/swift-DEVELOPMENT-SNAPSHOT-2021-11-20-a.xctoolchain
```

Comment:
I mean, no, we cannot hardcode the script to a specific path -- it should ask for the toolchain to be set instead for example -- require the TOOLCHAIN var exists.

Reply:
Something like this....

if  [[ ! -d "$docc_source_path" ]]; then 
  cd /Library/Developer/Toolchains/swift-DEVELOPMENT-SNAPSHOT-2021-11-20-a.xctoolchain
else
  echo "TOOLCHAIN var exists."

Reply:
@shilpeegupta14 - I suspect what you want to do is check to see if the environment variable TOOLCHAIN is already set, and if so - use it. For a linux platform with a nightly swift pre-installed and already added to the relevant paths (such as a docker build from a nightly swift toolchain release), then you won't need to prefix the commands with TOOLCHAIN, as they *should* be already on the path (I haven't double-checked that hypothesis though).

In either case, you'll want to likely set the `DOCC_HTML_DIR` environment variable so that `docc`, however you invoke it, knows where to find its HTML goodies it wants to combine/link with the output.

[StackOverflow has a nice reference](https://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash) to this otherwise awkward problem of determining if an environment variable is already set, which amounts to doing something like:
`if [ -z ${var+x} ]; then echo "var is unset"; else echo "var is set to '$var'"; fi`

Translating that to "require an environment TOOLCHAIN to be set in order to proceed" might be:

`if [ -z ${TOOLCHAIN+x} ]; then echo "TOOLCHAIN is unset"; exit(1); fi`

Then from there, build up the `DOCC_HTML_DIR`:

`export DOCC_HTML_DIR=${TOOLCHAIN}/usr/share/docc/render/`